src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Ball ball = new Lottery().getRandomBall();
```

Comment:
don't create a new lottery on each iteration
```suggestion
            Ball ball = new Lottery().getRandomBall();
```